src/clients/db/Repository.ts

```diff
@@ -47,8 +48,15 @@ export default class Repository<T extends AbstractDBModel> {
   }
 
   async save(entity: T): Promise<T> {
+    const currentTime = getTime(new Date())
+
     const { id, rev, ...valuesToSave } = entity
-    const savedEntity = await this.db.put({ _id: getTime(new Date()).toString(), ...valuesToSave })
+    const savedEntity = await this.db.put({
+      _id: uuidv4(),
+      ...valuesToSave,
+      createdDate: currentTime,
```

Comment:
I think we can use what is the standard defacto for naming these fields:
- `createdDate` -> `createdAt`
- `lastUpdatedDate` ->`updatedAt`

And use ISOString as format, as I already pointed out.